enhancements/kube-apiserver/bound-sa-tokens.md

```diff
@@ -0,0 +1,153 @@
+---
+title: bound-service-account-tokens
+authors:
+  - "@marun"
+reviewers:
+  - "@deads2k"
+  - "@mfojtik"
+  - "@stlaz"
+  - "@sttts"
+approvers:
+  - TBD
+creation-date: 2019-11-28
+last-updated: 2019-11-28
+status: provisional
+see-also:
+  - "https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/bound-service-account-tokens.md"
+  - "https://docs.google.com/document/d/1XcOsEv4jO9P1QQHn-tOnC80oMyCm85hGA6LqHRfjTgo/edit?ts=5ddb86c1"
+  - "https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/"
+  - "https://thenewstack.io/no-more-forever-tokens-changes-in-identity-management-for-kubernetes/"
+  - "https://jpweber.io/blog/a-look-at-tokenrequest-api/"
+replaces:
+superseded-by:
+---
+
+# Bound Service Account Tokens
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+- Is it necessary/desirable to allow `service-account-max-token-expiration` to be configured?
+
+## Summary
+
+Enable the optional use of bound service account tokens via volume projection and the
```

Comment:
links to the APIs would be nice.

Reply:
Done. Note that the best docs on both of these is arguably the jpweber.io link in `see-also`